lib/rspec/rails/fixture_support.rb

```diff
@@ -29,6 +29,27 @@ module FixtureSupport
           end
           self.use_instantiated_fixtures  = RSpec.configuration.use_instantiated_fixtures
           fixtures RSpec.configuration.global_fixtures if RSpec.configuration.global_fixtures
+
+          def self.fixtures(*args)
+            orig_methods = private_instance_methods
+            super.tap do
+              new_methods = private_instance_methods - orig_methods
+              new_methods.each do |method_name|
+                proxy_method_warning_if_called_in_before_context_scope(method_name)
+              end
+            end
```

Comment:
Could we make this a name module (I'm ok with dynamically creating it) so that it's clear from the ancestor chain we've inserted something?